README.md

```diff
@@ -51,6 +51,24 @@ Once approved, a feature should be fully implemented in [Roslyn](https://github.
 
 **DISCLAIMER**: An active proposal is under active consideration for inclusion into a future version of the C# programming language but is not in any way guaranteed to ultimately be included in the next or any version of the language. A proposal may be postponed or rejected at any time during any phase of the above process based on feedback from the design team, community, code reviewers, or testing.
 
+### Milestones
+
+We have a few different milestones for issues on the repo:
+* [Working Set](https://github.com/dotnet/csharplang/milestone/19) is the set of proposals that the LDT is currently actively working on. Not
+everything in this milestone will make the next version of C#, but it will get design time during the upcoming release.
+* [Backlog](https://github.com/dotnet/csharplang/milestone/10) is the set of proposals that members of the LDT have championed, but are not
+actively working on. While discussion and ideas from the community are welcomed on these proposals, the cost of the design work and
+implementation review on these features are too high for us to consider community implementation until we are ready for it.
+* [Any Time](https://github.com/dotnet/csharplang/milestone/14) is the set of proposals that members of the LDT have championed, but are not
```

Comment:
Same again; these went through LDM triage, and LDM decided to take them if someone does the work (and they approve of the final design)